_src/css/_base.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+    display: flex;
+    width: 100%;
+}
+
+.container__image {
```

Comment:
@felipprodrigues remember you are using Sass so you can take the advantage of their features like nesting.

In this case you don't need to repeat `container` everytime so you can just replace with `&__` as a child. 💯